pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -310,6 +314,16 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 		return nil, err
 	}
 
+	// initialize DRA manager
+	if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.DynamicResourceAllocation) {
+		klog.InfoS("Creating Dynamic Resource Allocation (DRA) manager")
+		cm.draManager, err = dra.NewManagerImpl(machineInfo.Topology, cm.topologyManager, kubeClient)
```

Comment:
Based on our discussion this morning it seems like we may want to remove all of the TopologyManager interfaces fromm this to avoid confusion.

Reply:
done in this PR: https://github.com/pohly/kubernetes/pull/22